site/src/pages/CreateWorkspacePage/permissions.ts

```diff
@@ -0,0 +1,16 @@
+export const createWorkspaceChecks = (organizationId: string) =>
```

Comment:
I'm guessing that these types are coming from XState. Do we have any files that have some of these types defined for when we eventually remove the library? One of my worries is that once we remove XState, our codebase will still be partly influenced by how it sets things up, but we won't have the type definitions

Reply:
Nops, there is no types coming from `XState` here 🤔

Reply:
Oh, I just assumed that they were, because I've seen a few objects structured with these `object` and `action` properties around the codebase, and I know "action" is a state machine term

Reply:
No it is not, it is related to our internal RBAC system 😁